.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
 rvm:
 - 2.7
+- '3.0'
```

Comment:
Explicitly use a string here. This avoids being interpreted as the numeric type `3.0` which gets simplified to `3`.